packages/app-desktop/gui/MasterPasswordDialog/Dialog.tsx

```
@@ -177,11 +177,16 @@ export default function(props: Props) {
 							onChange={onPasswordChange1}
 						/>
 						{needToRepeatPassword && (
+							<>
 							<LabelledPasswordInput
```

ESLint

[error] Expected indentation of 8 tabs but found 7.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 8 tabs but found 7.

(indent)